sdk/core/azure-core-management/pom.xml

```diff
@@ -59,7 +59,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.12.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+      <version>1.13.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core;current} -->
```

Comment:
Should this be a dependency as core-management is not released as frequently as core is released?
https://mvnrepository.com/artifact/com.azure/azure-core-management

Reply:
Maybe we can use dependency after this release (hope we do not need to update it for some time, just a hope).

Using dependency as default risks the case that people forget to pump it when do need the release (you can see last release uses core 1.11 instead of 1.12).

Reply:
Just to clarify, is this a recommendation to use `current` or `dependency`?

Reply:
My idea is if core-management is going to be released in next release cycle, it SHOULD be current.

My guess on why it was dependency before is to facilitate release of other core libs when core-management does not need a new release.